packages/manager/apps/telecom/src/app/telecom/pack/migration/pack-migration.routing.js

```diff
@@ -0,0 +1,13 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('telecom.packs.pack.migration', {
+    url: '/migration',
+    views: {
+      'packView@telecom.packs': 'packMigration',
+    },
+    resolve: {
+      packName: /* @ngInject */ ($transition$) =>
```

Comment:
No need to inject it if it is declared in parent route. Also if it is not, it is better to inject it so `packName` can be shared across all `telecom.packs.pack` children state